forge-game/src/main/java/forge/game/ability/AbilityUtils.java

```diff
@@ -1997,7 +1997,7 @@ public static int xCount(Card c, final String s, final CardTraitBase ctb) {
             return doXMath(calculateAmount(c, sq[!isUnlinkedFromCastSA(ctb, c) && c.getKickerMagnitude() > 0 ? 1 : 2], ctb), expr, c, ctb);
         }
         if (sq[0].startsWith("Escaped")) {
-            return doXMath(calculateAmount(c, sq[!isUnlinkedFromCastSA(ctb, c) && c.getCastSA() != null && c.getCastSA().isEscape() ? 1 : 2], ctb), expr, c, ctb);
```

Comment:
Escaped should be linked the to Escape Cost?

Reply:
I don't think so:
> 702.138b. A spell or permanent "escaped" if that spell or the spell that became that permanent as it resolved was cast from a graveyard with an escape ability.

Rulings on relevant cards agree with that:
>  Uro’s first ability causes you to sacrifice it if you didn’t cast it, or if it was cast using any permission other than an escape ability.